test/lib/tailwindcss/commands_test.rb

```diff
@@ -33,6 +33,34 @@ def setup
     end
   end
 
+  test ".compile_command debug environment variable" do
+    begin
+      Rails.stub(:root, File) do # Rails.root won't work in this test suite
```

Comment:
If `debug: true` is passed to `compile_command` but `TAILWINDCSS_DEBUG` is unset, which should take precedence? I *think* the environment variable probably should, but that's a weakly held opinion.

Regardless of which way you want it, I'd like test coverage for that case.

Reply:
Since the argument `debug` is `false` by default, I cannot test if the value was explicitly set to `false` without changing the methods signature. I would rather prefer having the environment variable take precedence, as we can check if it was explicitly set in the `ENV` hash